<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QMainWindow" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
        background-color: &quot;#363636&quot;;
}

QCustomPlot {
	border-radius: 8px;
}

QPushButton {
        border-radius: 8px;
        color: white;
        background-color: grey;
        font-size: 14px;
		padding: 5px;
        font-weight: bold;
}

QPushButton:hover:!pressed {
        border: 4px solid white;
}

QLabel {
	color: white;
}

QLineEdit {
        border: 1px solid white;
        border-radius: 8px;
        background-color: transparent;
        font-size: 16px;
		padding: 2px;
        color: white;
        font-weight: bold;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="10,1">
    <item>
     <widget class="QCustomPlot" name="plot" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>fromX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fromXEdit">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>toX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="toXEdit">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>fromY</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fromYEdit">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>toY</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="toYEdit">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="redrawBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: &quot;#a2a2d0&quot;;
	padding: 16px;
	margin: 2px;
	font-size: 16px;
}

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>Draw</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
